README.md

```diff
@@ -5,3 +5,9 @@ Wiki is [available online](https://github.com/OpenSC/OpenSC/wiki)
 Please take a look at the documentation before trying to use OpenSC.
 
 [![Travis CI Build Status](https://travis-ci.org/OpenSC/OpenSC.svg)](https://travis-ci.org/OpenSC/OpenSC/branches) [![AppVeyor CI Build Status](https://ci.appveyor.com/api/projects/status/github/OpenSC/OpenSC?branch=master&svg=true)](https://ci.appveyor.com/project/LudovicRousseau/OpenSC/branch/master) [![Coverity Scan Status](https://scan.coverity.com/projects/4026/badge.svg)](https://scan.coverity.com/projects/4026)
+
+Build and test status of specific cards:
+
+| Cards                 | Status |
+|-----------------------|--------|
+| CAC and Coolkey       | [![CAC and Coolkey](https://gitlab.com/redhat-sectech/OpenSC/badges/master/build.svg)](https://gitlab.com/redhat-sectech/OpenSC/pipelines) |
```

Comment:
I think we should not include Redhat's test status here. You could put that into the OpenSC wiki, if you like. For example, you could add that to https://github.com/OpenSC/OpenSC/wiki/Smart-Card-Testing. Some cards also have a dedicated wiki page, maybe that would also be an option for CAC/Coolkey.

Reply:
OK, I will leave this out.

Reply:
@frankmorgner the issue I see is that there is no other automated test, thus including something is better than nothing. Nevertheless, the Smart card testing page seems to be focused on manual testing per release, while this badge is for the master. I am not sure whether anyone would notice any breakage if that's included in the wiki.

Reply:
@CardContact do you also have some automated test suite we could include into opensc in addition to ours?